### PR TITLE
Add OpenFF ESP Industry Benchmark Set v1.1

### PR DESCRIPTION
**Pull request by SimonBoothroyd**
## New Submission Checklist

- [X] Created a new folder in the submissions directory containing the dataset
- [X] Added `README.md` describing the dataset see [here](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-6-supplemental-2) for examples
- [X] All files used to produce the dataset are included with a description
- [X] Dataset follows the QCSubmit schema defined for [Datasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L340), [OptimizationDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1062) and [TorsionDriveDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1225)
- [x] Dataset filename matches pattern `dataset*.json`; may feature a compression extension, such as `.bz2`
- [x] A PDF depicting the molecules is attached, in the case of torsiondrives this should include the highlighting of the central bond, this can be done automatically using [qcsubmit](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L854). 
- [x] QCSubmit validation passed
- [ ] Made a new dataset entry in the mapping table in repository `README.md`
- [x] Ready to submit!

## Compute Expansion Checklist

- [ ] Compute expansion filename matches pattern `compute*.json`; may feature a compression extension, such as `.bz2`
- [ ] QCSubmit validation passed
- [ ] Ready to submit!

### CODE REVIEW

**Comment by jthorton** [comment]
@SimonBoothroyd thanks, I have added the MBIS flag and our other defaults such as WBO as we may also need these for a gnn ? if not feel free to remove them apart from MBIS of course :)

File: submissions/2022-02-02-OpenFF-ESP-Industry-Benchmark-Set-v1.1-single-point/generate.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        base_molecule._conformers = [m.conformers[0] for _, m in records]
+
+        dataset.add_molecule(
+            index=base_molecule.to_smiles(
+                isomeric=True, explicit_hydrogens=False, mapped=False
+            ),
+            molecule=base_molecule,
+            attributes=MoleculeAttributes.from_openff_molecule(base_molecule),
+            extras=base_record.extras,
+            keywords=record.keywords,
```
Comment (jthorton) [inline submissions/2022-02-02-OpenFF-ESP-Industry-Benchmark-Set-v1.1-single-point/generate.py:80]:
```suggestion
            keywords=base_record.keywords,
```

**Comment by SimonBoothroyd** [comment]
@jthorton this seems to be using the default QC spec instead of the custom HF one specified, can you double check if I'm missing something here?

**Comment by jthorton** [comment]
@SimonBoothroyd I pulled down the dataset in the PR and it is showing one spec which is the OpenFF default so it looks like the spec was not updated correctly.

File: submissions/2022-02-02-OpenFF-ESP-Industry-Benchmark-Set-v1.1-single-point/generate.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        driver=DriverEnum.energy,
+        metadata=Metadata(
+            submitter="simonboothroyd",
+            long_description_url=(
+                "https://github.com/openforcefield/qca-dataset-submission/tree/master/"
+                "submissions/"
+                "2022-02-02-OpenFF-ESP-Industry-Benchmark-Set-v1.1-single-point"
+            )
+        ),
+        qc_specs={
```
Comment (jthorton) [inline submissions/2022-02-02-OpenFF-ESP-Industry-Benchmark-Set-v1.1-single-point/generate.py:41]:
```suggestion
        qc_specifications={
```

Comment (jthorton) [reply submissions/2022-02-02-OpenFF-ESP-Industry-Benchmark-Set-v1.1-single-point/generate.py:41]: @SimonBoothroyd it looks like the field name is incorrect.

Comment (SimonBoothroyd) [reply submissions/2022-02-02-OpenFF-ESP-Industry-Benchmark-Set-v1.1-single-point/generate.py]: Ah great catch!

**Comment by dotsdl** [comment]
Hmmm...not clear to me why the `dataset.json.xz` file isn't going to LFS instead of getting committed and pushed here. Trying to think of ways to characterize this further before we proceed.

**Comment by SimonBoothroyd** [comment]
Sounds good - let me know if there's anything to do to help get this merged as this a a slight block on some science I want to get started 🙂

**Pull request closed without merging**